lib/spoom/deadcode/plugins/active_job.rb

```diff
@@ -5,6 +5,7 @@ module Spoom
   module Deadcode
     module Plugins
       class ActiveJob < Base
+        ignore_classes_inheriting_from("ActiveJob::TestCase")
```

Comment:
Isn't `ActiveJob::TestCase` a subclass of `ActiveSupport::TestCase` (https://github.com/rails/rails/blob/main/activejob/lib/active_job/test_case.rb#L6)? The `ActiveSupport` plugin should be catching this so there is another kind of bug at play here. I don't think this is the right fix.

Reply:
>Isn't ActiveJob::TestCase a subclass of ActiveSupport::TestCase

It definitely is but since it wasn't working I just assumed that "for reasons" we only look at the direct parent without trying to analyze the whole inheritance chain.

But if we expect `ignore_classes_inheriting_from` to apply to the whole chain then I totally agree we should be looking into keeping only `ActiveSupport::TestCase` as ignored parent and fixing the lookup instead

>I wonder what would be the impact of also adding .rbi files to the Model

I can allocate some time to do that but since we are experiencing an issue right now and current version has a bug how would you feel about either merging this PR with explicit ignore of `ActiveJob::TestCase` or perhaps temporarily removing changes to `Minitest` plugin (bringing back ignore of `*Test` classes

Reply:
I think we can merge this. Though, since you know Rails can you ensure that we not missing other subclasses of `ActiveSupport::TestCase`?

I'll handle the RBI indexing exploration 👍